glob.go

```diff
@@ -301,7 +311,7 @@ func indexMatchedOpeningAlt(s string) int {
 
 // Returns true if the path exists
 func exists(fsys fs.FS, name string) bool {
-	if _, err := fs.Stat(fsys, name); err != nil {
+	if _, err := fs.Stat(fsys, name); os.IsNotExist(err) {
```

Comment:
Why did you make this change? The code was checking for any error to match behavior with [io/fs.Glob](https://cs.opensource.google/go/go/+/refs/tags/go1.18.3:src/io/fs/glob.go;l=43)

Reply:
I shouldn't have.I reverted this particular change